app/models/solicitation.rb

```diff
@@ -28,6 +28,7 @@ class Solicitation < ApplicationRecord
   #
   belongs_to :landing, primary_key: :slug, foreign_key: :landing_slug, inverse_of: :solicitations, optional: true
   has_many :diagnoses, inverse_of: :solicitation
+  has_many :feedbacks, as: :feedbackable
```

Comment:
`dependent: :destroy`, comme depuis les needs?
```suggestion
  has_many :feedbacks, dependent: :destroy, as: :feedbackable
```